04_numeronym/lokili/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var testSet = []struct {
+	input  []string
+	output []string
+}{
+	{[]string{"世界Thrives！"}, []string{"世8！"}},
+	{[]string{""}, []string{""}},
+	{[]string{"&《 44za  "}, []string{"&7 "}},
+	{[]string{"？will他 reach the airport at 10 am ?"}, []string{"？33?"}},
```

Comment:
You should also test your `numeronyms` function against input arrays of varying length, eg
`[]String{"", "abcd"}`
`[]String{}`